client/app/components/Input/InputError.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
```

Comment:
Let's write a test for this! Check out the `__tests__` folder in Input for examples. Let me know if you want to pair on this :)